src/app/component-status/component-status.component.ts

```diff
@@ -52,30 +48,75 @@ export class ComponentStatusComponent implements OnInit {
     this.searchTerm$.next(this.searchTerm$.value);
   }
 
-  private initializeItems() {
-    if (this.githubStatusLoaded && this.githubComponentRequestsLoaded) {
-      this.sortedItems = this.sortItems(this.items);
-      this.isLoading = false;
-    }
+  private initializeGithubStatus() {
+    console.time('Initialize Github Status');
```

Comment:
Is there a reason behind us keeping this console."log" (time metrics), or should they be removed (I'm fully aware they're not adding any harm, just logging to the console.

Reply:
Nope, they were just for testing how long time it actually takes :)
I've removed them.